go/vt/vtctl/workflow/server.go

```diff
@@ -3097,7 +3099,7 @@ func (s *Server) WorkflowSwitchTraffic(ctx context.Context, req *vtctldatapb.Wor
 }
 
 // switchReads is a generic way of switching read traffic for a workflow.
-func (s *Server) switchReads(ctx context.Context, req *vtctldatapb.WorkflowSwitchTrafficRequest, ts *trafficSwitcher, state *State, timeout time.Duration, cancel bool, direction TrafficSwitchDirection) (*[]string, error) {
```

Comment:
were `timeout` and `cancel` unused? I'm surprised that the linter did not catch this.

Reply:
Yes, they were (and highlighted as such in VS Code). I took this opportunity to remove them.